### ISSUE TITLE
cannot startup with version 1.1.1

### ISSUE THREAD

**Issue by cjl111** [spring-projects-experimental/spring-cloud-sleuth-otel#153]
I can't find the appropriate version of sleuth-otel and spring-cloud. Now the startup will report an error. Can you provide a correct version comparison

![image](https://user-images.githubusercontent.com/38237837/218994804-4c3ddb52-8712-4092-939d-3b1e240b60a7.png)

![image](https://user-images.githubusercontent.com/38237837/218994705-3c75921f-e8ed-41ad-83dc-ab9454195914.png)

Describe the bug
When starting my Spring Boot app with Spring Boot 2.6.4, Spring Cloud 2021.0.1 and Spring Cloud Sleuth OTel 1.1.1, I get this exception on startup:
`Caused by: java.lang.ClassNotFoundException: io.opentelemetry.exporter.otlp.internal.grpc.GrpcExporter
	at java.net.URLClassLoader.findClass(URLClassLoader.java:387) ~[na:1.8.0_361]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:418) ~[na:1.8.0_361]
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:355) ~[na:1.8.0_361]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[na:1.8.0_361]
	... 118 common frames omitted`
![image](https://user-images.githubusercontent.com/38237837/218997016-597e9cbc-d8f8-4429-a366-3bde9ffac0bb.png)

Sample
Here's a reproducer project with the setup that provokes the above error: https://github.com/cjl111/otel-demo

**Comment by marcingrzejszczak** [comment]
You have wrong version of otlp exporter on your classpath I guess.

**Comment by cjl111** [comment]
> You have wrong version of otlp exporter on your classpath I guess.

Where to find the right match version?

**Comment by marcingrzejszczak** [comment]
If you're using Sleuth OTel 1.1.2 then these are the versions that we provide + we're using the OTel's BOM so you shouldn't need to provide a version for the jar. It should come from the BOM

```xml
<opentelemetry.version>1.22.0</opentelemetry.version>
		<opentelemetry-alpha.version>1.22.0-alpha</opentelemetry-alpha.version>
		<opentelemetry-instrumentation.version>1.22.0</opentelemetry-instrumentation.version>
		<opentelemetry-instrumentation-alpha.version>1.22.0-alpha</opentelemetry-instrumentation-alpha.version>
		<opentelemetry-proto.version>0.11.0-alpha</opentelemetry-proto.version>
```

**Comment by cjl111** [comment]
> If you're using Sleuth OTel 1.1.2 then these are the versions that we provide + we're using the OTel's BOM so you shouldn't need to provide a version for the jar. It should come from the BOM
> 
> ```
> <opentelemetry.version>1.22.0</opentelemetry.version>
> 		<opentelemetry-alpha.version>1.22.0-alpha</opentelemetry-alpha.version>
> 		<opentelemetry-instrumentation.version>1.22.0</opentelemetry-instrumentation.version>
> 		<opentelemetry-instrumentation-alpha.version>1.22.0-alpha</opentelemetry-instrumentation-alpha.version>
> 		<opentelemetry-proto.version>0.11.0-alpha</opentelemetry-proto.version>
> ```

I can't load 1.1.2. I don't know why. I can only download 1.1.0 and 1.1.1

**Comment by marcingrzejszczak** [comment]
It's there https://search.maven.org/artifact/org.springframework.cloud/spring-cloud-sleuth-otel-dependencies/1.1.2/pom - I don't know why you can't download it

**Comment by cjl111** [comment]
> It's there https://search.maven.org/artifact/org.springframework.cloud/spring-cloud-sleuth-otel-dependencies/1.1.2/pom - I don't know why you can't download it

Thank you very much. When I reset the maven repository image, I can download version 1.1.2. Now I can start normally
But there is a new problem. I found that my log has not been uploaded to third-party monitoring
The old version used yesterday can be collected normally.
![image](https://user-images.githubusercontent.com/38237837/219320039-92c4f72f-b94c-4ce4-b6ac-840c2bd79e33.png)

But when I upgraded to Sleuth OTel 1.1.2, I didn't know which link had the problem without being collected
My dependent version
![image](https://user-images.githubusercontent.com/38237837/219320699-846f3819-1928-48c6-8b73-06afc4707a43.png)

Log display is normal
![image](https://user-images.githubusercontent.com/38237837/219320317-b09b0798-1a04-4207-bf58-6986a52503fd.png)

Where to check this situation
